src/main/java/io/voucherify/client/api/VoucherifyApi.java

```diff
@@ -489,4 +490,10 @@ Call<RedeemVoucherResponse> redeemLoyaltyReward(
   @GET("loyalties/members/{memberId}/tiers")
   Call<MembersLoyaltyTierResponse> getMembersLoyaltyTier(
           @Path("memberId") String memberId);
+
+  @GET("loyalties/members/{memberId}/activities")
+  Call<GetActivitiesResponse> getLoyaltyActivities(@Path("memberId") String memberId);
```

Comment:
maybe we should choose one name for this action from `GetMemberActivities` or `GetLoyaltyActivities`? In docs it is described as `GetMemberActivities`  (there's also a method `getMemberActivities` in JS SDK) 
but maybe there is a specific reason for such a name, if so, it's not an issue for me :D